### ISSUE TITLE
issue with nipypye and network >2.0

### ISSUE THREAD

**Issue by grandjeanlab** [IBT-FMI/SAMRI#69]
I tried running test_script.py. 
Had following error message using python 3.5 on ubuntu 16.04: 
  File "/usr/local/lib/python3.5/dist-packages/nipype/pipeline/engine/workflows.py", line 963, in _get_dot
    for subnode in self._graph.successors_iter(node):
AttributeError: 'DiGraph' object has no attribute 'successors_iter'

I downgraded to networkx 1.11 and it fixed it.

**Comment by TheChymera** [comment]
SAMRI does not call networkx directly, so this is a nipype rather than a SAMRI issue.
Still, please tell me the nipype version or if it's not an explicit version the exact commit (the long string of letters and numbers, e.g in [this link](https://github.com/nipy/nipype/tree/7055313c91de52bf65518284b42df446466fe581)), so I can fix this in the Gentoo package manager.

If you are using another package manager, I suggest you contact them with this detail, if you manage your software manually, there is sadly nothing you can do. You will have to make a note somewhere for any future installations, that the nipype version you use requires `=networkx-1.11`, this is the sort of knowledge which package managers integrate and distribute.

**Comment by grandjeanlab** [comment]
I use nipype 1.0.0.dev0 corresponding to tree 7055313c91de52bf65518284b42df446466fe581 and manange my packages manually so far.

**Comment by TheChymera** [comment]
At any rate this seems to have been solved, tests pass here with the following versions (both newer than in this report):

```
chymera@zenhost ~ $ eix networkx
[I] dev-python/networkx
     Available versions:  1.11 1.11-r1 (~)2.1 {doc examples extras pandas scipy test xml yaml PYTHON_TARGETS="python2_7 python3_4 python3_5 python3_6"}
     Installed versions:  2.1(11:33:24 30/07/18)(xml -examples -extras -pandas -scipy -test -yaml PYTHON_TARGETS="python2_7 python3_6 -python3_5")
     Homepage:            http://networkx.github.io/ https://github.com/networkx/networkx
     Description:         Python tools to manipulate graphs and complex networks

chymera@zenhost ~ $ eix nipype
[I] sci-libs/nipype [1]
     Available versions:  (~)0.12.0 (~)0.12.1 (~)0.14.0_pre20170830 (~)1.1.0 (~)1.1.1 **9999 {test PYTHON_TARGETS="python2_7 python3_4 python3_5 python3_6"}
     Installed versions:  1.1.1(16:01:54 16/01/19)(-test PYTHON_TARGETS="python2_7 python3_6 -python3_4 -python3_5")
     Homepage:            http://nipy.sourceforge.net/nipype/
     Description:         Neuroimaging in Python: Pipelines and Interfaces

[1] "science" /var/repos/science
```
